packages/platform-sdk/test/integration-tests/order/order.test.ts

```diff
@@ -0,0 +1,227 @@
+    const getCart = await apiRoot
+      .carts()
+      .withId({ ID: order.body.cart.id })
+      .get()
+      .execute()
+    await deleteCart(getCart)
+    await deleteProduct(product)
+    await deleteProductType(productType)
+    await deleteTaxCategory(taxCategory)
+    await deleteCategory(category)
+  })
+
+  it('should get a order by order number', async () => {
+    const category = await createCategory()
+    const taxCategory = await createTaxCategory()
+    const productType = await createProductType(productTypeDraftForProduct)
+    const productDraft = await createProductDraft(
+      category,
+      taxCategory,
+      productType,
+      true
+    )
+    const product = await createProduct(productDraft)
+    const cart = await createCart()
+    const order = await createOrder(cart, product)
+
+    const getOrder = await apiRoot
+      .orders()
+      .withOrderNumber({ orderNumber: order.body.orderNumber })
+      .get()
+      .execute()
+
+    expect(getOrder).not.toBe(null)
+    expect(getOrder.body.id).toEqual(order.body.id)
+
+    await deleteOrder(order)
+    const getCart = await apiRoot
+      .carts()
+      .withId({ ID: order.body.cart.id })
+      .get()
+      .execute()
+    await deleteCart(getCart)
+    await deleteProduct(product)
+    await deleteProductType(productType)
+    await deleteTaxCategory(taxCategory)
+    await deleteCategory(category)
+  })
+
+  it('should update a order', async () => {
+    const category = await createCategory()
+    const taxCategory = await createTaxCategory()
+    const productType = await createProductType(productTypeDraftForProduct)
+    const productDraft = await createProductDraft(
+      category,
+      taxCategory,
+      productType,
+      true
+    )
+    const product = await createProduct(productDraft)
+    const cart = await createCart()
+    const order = await createOrder(cart, product)
+
+    const updateOrder = await apiRoot
+      .orders()
+      .withId({ ID: order.body.id })
+      .post({
+        body: {
+          version: order.body.version,
+          actions: [
+            {
+              action: 'setBillingAddress',
+              address: {
+                country: 'DE',
+                state: 'Berlin',
+              },
+            },
+            {
+              action: 'changeOrderState',
+              orderState: 'Confirmed',
+            },
+          ],
+        },
+      })
+      .execute()
+    expect(updateOrder.body.version).not.toBe(order.body.version)
+    expect(updateOrder.statusCode).toEqual(200)
+
+    await deleteOrder(updateOrder)
+    const getCart = await apiRoot
+      .carts()
+      .withId({ ID: updateOrder.body.cart.id })
+      .get()
+      .execute()
+    await deleteCart(getCart)
+    await deleteProduct(product)
+    await deleteProductType(productType)
+    await deleteTaxCategory(taxCategory)
+    await deleteCategory(category)
+  })
+
+  it('should search a order', async () => {
```

Comment:
I think it's better to disable this test with a comment why. Right now this test is a bit confusing, because it does not do the search, even though the name says so.